src/renderer/src/pages/game-details/seeders-and-peers/useMagnetData.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from "react";
```

Comment:
Do you really think that it's necessary to create a hook that is going to be used in a single place?

Reply:
I prefer to organize components with custom hooks and utilities to keep them focused on rendering, making the code more modular and maintainable. What do you think?